input-output-hk/daedalus
# [CBR-391] launcher-config: use a relative node DB path on Windows

`cardano-launcher` got a new "workingDir" setting. On Windows, provide a relative path for the node DB. This works around unicode bugs in rocksdb.

https://iohk.myjetbrains.com/youtrack/issue/CBR-391

## QA Steps

1. Log in to Windows as user "ŮŝĕŗŃåɱĕ".
2. Start Daedalus.
3. Check that cardano-node.exe has not failed with some c_rocksdb_open error.

## Do not merge

Use this branch for testing the bugfix. Do not merge until input-output-hk/cardano-sl#3465 is merged. After that is merged, update `cardano-sl-src.json` in this branch.
